src/backend/src/controllers/change-requests.controllers.ts

```diff
@@ -42,7 +42,7 @@ export const reviewChangeRequest = async (req: Request, res: Response) => {
   const reviewer = await prisma.user.findUnique({ where: { userId: reviewerId } });
   if (!reviewer) return res.status(404).json({ message: `User with id #${reviewerId} not found` });
   if (reviewer.role === Role.GUEST || reviewer.role === Role.MEMBER)
-    return res.status(401).json({ message: 'Access Denied' });
```

Comment:
Are these errors exposed directly to the user, or only the frontend programmers? 

If they are, Role X does not have access to do this might be more informative. It's my understanding that these messages are eaten by the frontend and would never surface to the user, so this is a moot point.

Reply:
so i think that my changes to the axios file should make these error messages visible so yeah i should make some tickets to make these better